addons/web/static/lib/py.js/lib/py_extras.js

```diff
@@ -556,7 +556,7 @@ datetime.datetime = py.type('datetime', null, {
             }));
     },
     to_utc: function () {
-        var d = new Date(this.year, this.month, this.day, this.hour, this.minute, this.second);
+        var d = new Date(this.year, this.month -1, this.day, this.hour, this.minute, this.second);
```

Comment:
Hello, I'm not sure why we need to edit the month here, is a special calculation related to `to_utc`?

Reply:
You can have a look to https://github.com/odoo/odoo/pull/103353/commits/1b83d21675c393078830b482980d94e0aed555e0

Reply:
I am going to fix this in 13.0  and add a test. You can remove it for now. If you don't do that and work in the timezone, you will see that the dates generated in python or js are different for this month :-)

Reply:
Thanks for the explaination! :smile: